rest/oidc_api.go

```diff
@@ -82,7 +107,13 @@ func (h *handler) handleOIDCCommon() (redirectURLString string, err error) {
 	}
 
 	var redirectURL *url.URL
-	state := ""
+	var state string
```

Comment:
I think leaving the initialization as it was (state := "") is a more readable approach, and makes it clear that empty string is the intended usage when provider.DisableCallbackState=true.